bids/analysis/transformations/compute.py

```diff
@@ -63,15 +69,19 @@ def _transform(self, var, model='spm', derivative=False, dispersion=False,
         elif model != 'fir':
             raise ValueError("Model must be one of 'spm', 'glover', or 'fir'.")
 
-        # Minimum interval between event onsets/duration
-        # Used to compute oversampling factor to prevent information loss
-        unique_onsets = np.unique(np.sort(df.onset))
-        if len(unique_onsets) > 1:
-            min_interval = min(np.ediff1d(unique_onsets).min(),
-                               df.duration.min())
-            oversampling = np.ceil(2*(1 / (min_interval * sampling_rate)))
```

Comment:
@tyarkoni I did not fully preserve this behavior, so this will break the "dynamic oversampling" tests.

https://github.com/bids-standard/pybids/blob/7d2bade83e92548363a1d0ce2cdce72519479042/bids/analysis/tests/test_transformations.py#L75-L93

Can you check whether the new behavior is reasonable, or if we need to cover other cases?

Reply:
Just ran a git blame, and realized this is @adelavega's work... Alejandro, could you have a look at this?